plugins/hydrate/hydrate.js

```diff
@@ -0,0 +1,32 @@
+const settings = require('./hydrate.json');
+
+module.exports = {
+  name: 'Hydrate',
+  description: 'Remind the streamer to hydrate',
+  chat: true,
+  event: false,
+  type: 5004,
+  command: 'hydrate',
+  permissions: [],
+  cooldown: 60,
+  settings: false,
+  credits: `Made by Rehkloos`,
+  execute(client) {
+
+    var THIRTY_MINUTES = 1000 * 60 * 30;
+
+    if (!settings.active) {
+      console.log('Please change active to true for command to work');
+      client.sendMessage('Streamer disabled this command');
+    } else {
+      let startDate = Date().toString();
+      console.log(`starting hydration messages at ${startDate}`);
+      client.sendMessage('Great, I will make sure the streamer stays hydrated! 💧 ');
+      var interval = setInterval(function() {
+        let dateTime = Date().toString();
+        console.log(`sending message at ${dateTime}`);
+        client.sendMessage('Streamer, I recommend you drink 240ml (8 oz) of water! I will remind you again in 30 minutes');
```

Comment:
we should calculate the TIME_BETWEEN_DRINKS using Date, to find out how long it is between the next drink, and 8 oz? every 3mins? wtf? you trying to drown the streamer?

Reply:
8 oz is half a bottle of water (16.9 are standard water bottles)